### DISCUSSION TITLE
Extreme lag on input thread

### DISCUSSION THREAD

**Discussion by Kuuuube** [ppy/osu#26822] [Q&A] [answered]
The input thread has incredibly large lag spikes frequency making the game unplayable. This occurs on all maps I tested (a random 20 or so). For this testing I am using a 1000hz mouse and keyboard.

The last time I played on lazer was on version 2023.1221.0-lazer. It did not have this issue.

Video of the problem (had to record on phone, obs caused too much additional lag):

https://github.com/ppy/osu/assets/61125188/06ea95a4-a064-4cd7-89c6-602f62c5dcd2

Skin:

https://kuuube.s-ul.eu/OMjjEy1T

Logs:

[1706632498.performance.log](https://github.com/ppy/osu/files/14101816/1706632498.performance.log)
[1706632498.runtime.log](https://github.com/ppy/osu/files/14101817/1706632498.runtime.log)
[1706632498.database.log](https://github.com/ppy/osu/files/14101818/1706632498.database.log)
[1706632498.network.log](https://github.com/ppy/osu/files/14101819/1706632498.network.log)

**Comment by peppy** [comment, marked as answer]
This is usually caused by another app on your system blocking the windows event loop. Please start by closing other apps one by one until you find the app which is causing the issue. And please let us know which app it is so we can try and diagnose further.

**Reply by Kuuuube** [reply]
Thank you for the reply. It seems that Microsoft PowerToys was causing the issue. 

Strangely, I can't seem to replicate the issue after starting powertoys again but I gave a reasonable amount of testing time inbetween closing each app and verified they were closed with task manager. It instantly worked after powertoys was off so I'm quite certain it was that.
